lang/csharp/src/apache/main/Avro.main.csproj

```diff
@@ -19,7 +19,7 @@
   <Import Project="../../../common.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netcoreapp2.1</TargetFrameworks>
```

Comment:
We can't until #773 is merged. See my comment [here](https://github.com/apache/avro/pull/792#issuecomment-579318452)

>> The CI is failing:

> In order to fix it, we would need to install the .NET SDK v3.1 on those CI machines. But I don't know how to do that. So instead, I dropped the netstandard2.1 target. We can easily add it back when the CI machines are updated for the latest .NET SDK.

Also see my note in the original comment of the PR:

> I needed to multi-target to netstandard2.1 in order to take advantage of the new APIs. I also added a target for netcoreapp2.1 because .NET Core 2.1 is still supported, so anyone using this library on 2.1 can take advantage of better performance.

Reply:
FYI - now that #798 is merged, I added the `netstandard2.1` target back. I also left the `netcoreapp2.1` target for the above reasons. Let me know what you think.